src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -62,12 +109,16 @@
 				if (resourceSource) {
 					labels['resource'] = 'vscode-web-workbench';
 					labels['error'] = 'LoadError';
+					gitpodMetricsReportError(new Error("LoadError"), { resource: labels['resource'], url: resourceSource })
 				} else {
 					labels['error'] = 'Unknown';
 				}
 				gitpodMetricsAddCounter('gitpod_supervisor_frontend_error_total', labels);
-			};
 
+				// TODO collect errors, we can capture resourceSource, error if possible with stack, message, name, and window URL
```

Comment:
@mustard-mh it is done?

Reply:
I'll clean some comments up by push commit to `gp-code/main`